pkg/planner/cardinality/selectivity.go

```diff
@@ -621,6 +671,34 @@ func getMaskAndRanges(ctx sessionctx.Context, exprs []expression.Expression, ran
 	return mask, ranges, false, nil
 }
 
+func getMaskAndSelectivityForMVIndex(
```

Comment:
Could you please add some example comments with the code, for example:
```
func getMaskAndSelectivityForMVIndex(
    ...
    remainedExprs []Expr // [(x member of j1), (y member of j2), (others)]
) {
    ....
    accessConds, _ := CollectFilters4MVIndex(ctx, remainedExprs, cols) // [(x member of j1), (y member of j2)]
    ....
}
```
Like this:
<img width="765" alt="image" src="https://github.com/pingcap/tidb/assets/7499936/3ebfe0fd-0717-4920-b2bb-27eae7634c70">

Reply:
I think this logic is not complicated, and the existing comments on `collectFilters4MVIndex()` and `buildPartialPaths4MVIndex()` are enough, so I added a reference to those comments.